test/e2e/network/networking.go

```diff
@@ -532,6 +532,9 @@ var _ = SIGDescribe("Networking", func() {
 			}
 			return false, nil
 		})
+		if err != nil {
+			e2essh.LogResult(result)
+		}			
```

Comment:
gofmt complains here
```
diff -u ./test/e2e/network/networking.go.orig ./test/e2e/network/networking.go
--- ./test/e2e/network/networking.go.orig	2020-10-08 18:22:34.196558025 +0000
+++ ./test/e2e/network/networking.go	2020-10-08 18:22:34.196558025 +0000
@@ -534,7 +534,7 @@
 		})
 		if err != nil {
 			e2essh.LogResult(result)
-		}			
+		}
 		framework.ExpectNoError(err, "kubelet did not recreate its iptables rules")
 	})
 })
```

Reply:
didn' t realize Dan already reported it sorry
https://github.com/kubernetes/kubernetes/pull/95378#issuecomment-705772125